snafu/dns_perf_wrapper/Readme.md

```diff
@@ -0,0 +1,39 @@
+## DNS Performance Workload Wrapper
+Wrapperf around [dnsperf](https://github.com/DNS-OARC/dnsperf) to measure throughput ( requests per second ) and latency metrics for Domain Name Service ( DNS ). The collected metrics are scraped to generate a json which can be indexed into Elasticsearch datastore.
+
+### Dependencies
+This wrapper requires a couple of dependencies as well as enabling copr repo to download the latest dnsperf package. For more details [see[(https://github.com/DNS-OARC/dnsperf/blob/master/README.md). It's recommended to build and run the [containerized version](Dockerfile) to avoid managing the installation on the host.
```

Comment:
The dependencies section is confusing me a bit. It might be useful to exclude mention of the copr repo as that's not the dependency, but where the dependency is currently being retrieved from (if I am following correctly)

I'm not sure if this section is noting the the dependencies of the constructed Dockerfile, or the wrapper. If it's the latter case, something like this might be a little easier to follow (and maintain):

- dnsperf
- python3 (may want to specify a specific version of Python, e.g. 3.8)
- etc...

Reply:
You are right @kurtismullins, copr repo is where the dependency is being retrieved from but it's a custom repo needed to download dns-perf, so it kind of acts as a dependency :-)

Will mention the dependencies which are mentioned in the dockerfile in the readme to make it clear like you suggested. Thanks.